faunadb-java-dsl/src/main/java/com/faunadb/client/types/Path.java

```diff
@@ -145,7 +146,9 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return Joiner.on("/").join(segments);
+    return segments.stream()
```

Comment:
`String.join("/", segments)` wouldn't be the same?

Reply:
Segments is not `List<String>`, it's `List<Segment>`.